src/React/Components/Login/login.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import Avatar from "@material-ui/core/Avatar";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import TextField from "@material-ui/core/TextField";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import Link from "@material-ui/core/Link";
+import Grid from "@material-ui/core/Grid";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import Container from "@material-ui/core/Container";
+import validator from "validator";
+import empty from "is-empty";
+
+const useStyles = makeStyles((theme) => ({
+    paper: {
+        marginTop: theme.spacing(8),
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+    },
+    avatar: {
+        margin: theme.spacing(1),
+        backgroundColor: theme.palette.secondary.main,
+    },
+    form: {
+        width: "100%",
+        marginTop: theme.spacing(1),
+    },
+    submit: {
+        margin: theme.spacing(3, 0, 2),
+    },
+}));
+
+export default function Login() {
+    const classes = useStyles();
+
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [allErrors, setAllErrors] = useState({});
+
+    function nonEmptyForm() {
+        return email.length > 0 && password.length > 0;
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault();
+        const errors = {};
+        errors.email = validator.isEmail(email)
+            ? ""
+            : "Please enter a valid email.";
+        errors.login =
+            email === "example@gmail.com" && password === "1234"
+                ? ""
+                : "Email or password is incorrect.";
+
+        setAllErrors(errors);
+
+        if (empty(errors.email) && empty(errors.login)) {
+            window.alert("You're in :)");
```

Comment:
A future PR will route here to a new page right?

Reply:
Yes exactly, this would take the user to an admin dashboard or something along the lines of that